fee_allocator/accounting/core_pools.py

```diff
@@ -0,0 +1,123 @@
+from __future__ import annotations
+from dataclasses import dataclass, field
+from web3 import Web3
+from typing import List, Dict, TYPE_CHECKING
+from decimal import Decimal
+from datetime import datetime
+
+from bal_tools.models import PoolSnapshot, TWAPResult
+from fee_allocator.accounting.interfaces import AbstractCorePool
+from fee_allocator.accounting.overrides import CorePoolOverride, overrides
+
+if TYPE_CHECKING:
+    from fee_allocator.accounting.chains import Chain
+
+
+@dataclass
+class CorePoolData:
+    pool_id: str
+    label: str
```

Comment:
what is label?  Maybe a comment and/or some kind of basic docs here?

Reply:
label is the pool symbol. i agree, should be more specific. i think renaming it to "symbol" will remove the ambiguity

Reply:
renamed in [cfb89b0](https://github.com/BalancerMaxis/protocol_fee_allocator_v2/pull/1/commits/cfb89b0b01c77ee457e51a237f1bdb674c7ffd49)